docs/headlampFAQ.md

```diff
@@ -0,0 +1,132 @@
+---
+Title: Frequently Asked Questions
+---
+
+# General Questions
+
+### Q: What is Headlamp and who is it for?
+
+**A:** Headlamp is an open source, extensible web interface designed to simplify the management of Kubernetes clusters. It serves as a visual management tool ideal for developers, DevOps professionals, and system administrators who are looking for an intuitive way to oversee cluster operations, deployments, and troubleshooting.
+
+### Q: On which platforms can I run Headlamp?
+
+**A:** Headlamp is platform-agnostic in terms of its web interface, which can be accessed through any modern web browser. For desktop use, there are specific versions tailored for Windows, macOS, and Linux, ensuring a wide range of compatibility for different users.

```

Comment:
When answering the 1st question, be sure to mention that though it's vendor agnostic, we do have a section in the docs with the tested platforms.